request_url.go

```diff
@@ -43,7 +43,11 @@ func createRequestUrl(method, host, path, key, secret, timestamp string, secure
 	params.Add("auth_signature", authSignature)
 
 	if host == "" {
-		host = "api.pusherapp.com"
+    if cluster != "" {
+      host = "api-"+cluster+".pusher.com"
```

Comment:
You could use `fmt.Sprintf("api-%s.pusher.com", cluster)` - which just formats and returns the string.